test-integration/test-server/pom.xml

```diff
@@ -11,7 +11,7 @@
     <packaging>war</packaging>
 
     <properties>
-        <version.infinispan.hotrod>10.0.1.Final</version.infinispan.hotrod>
+        <version.infinispan.hotrod>11.0.3.Final</version.infinispan.hotrod>
```

Comment:
master branch is based on Infinspan 12 version

Reply:
Fixed